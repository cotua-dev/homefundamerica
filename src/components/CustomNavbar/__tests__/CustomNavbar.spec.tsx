import React from 'react';
import { render, RenderResult } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import { CustomNavbar } from '..';

describe('CustomNavbar component', (): void => {
    test('renders', (): void => {
        const { container }: RenderResult = render(<CustomNavbar/>);

        expect(container.firstChild).toMatchSnapshot();
    });
});
