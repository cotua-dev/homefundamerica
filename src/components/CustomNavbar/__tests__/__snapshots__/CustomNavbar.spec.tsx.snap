// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CustomNavbar component renders 1`] = `
<header>
  <nav>
    <div>
      <svg
        aria-hidden="true"
        class="svg-inline--fa fa-bars fa-w-14 "
        data-icon="bars"
        data-prefix="fas"
        focusable="false"
        role="img"
        viewBox="0 0 448 512"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M16 132h416c8.837 0 16-7.163 16-16V76c0-8.837-7.163-16-16-16H16C7.163 60 0 67.163 0 76v40c0 8.837 7.163 16 16 16zm0 160h416c8.837 0 16-7.163 16-16v-40c0-8.837-7.163-16-16-16H16c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16zm0 160h416c8.837 0 16-7.163 16-16v-40c0-8.837-7.163-16-16-16H16c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16z"
          fill="currentColor"
        />
      </svg>
    </div>
    <a
      href="/"
    >
      <img
        alt="Mbanc Logo"
      />
    </a>
    <div>
      <strong>
        Residential
        <svg
          aria-hidden="true"
          class="svg-inline--fa fa-caret-down fa-w-10 "
          data-icon="caret-down"
          data-prefix="fas"
          focusable="false"
          role="img"
          viewBox="0 0 320 512"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M31.3 192h257.3c17.8 0 26.7 21.5 14.1 34.1L174.1 354.8c-7.8 7.8-20.5 7.8-28.3 0L17.2 226.1C4.6 213.5 13.5 192 31.3 192z"
            fill="currentColor"
          />
        </svg>
      </strong>
    </div>
    <a
      href="/conventional"
    >
      <button
        type="button"
      >
        Apply Now
      </button>
    </a>
  </nav>
</header>
`;
